superchain/superchain_test.go

```diff
@@ -294,7 +294,12 @@ func TestCanyonTimestampOnBlockBoundary(t *testing.T) {
 			chainCfg := OPChains[id]
 			canyonOffset := ct - chainCfg.Genesis.L2Time
 			// Block time is hardcoded in op-node/rollup/superchain.go
-			if canyonOffset%2 != 0 {
+			blockTime := 2
+			// Zora Base has a 10s block time.
```

Comment:
```suggestion
			// Zora Goerli has a 10s block time.
```

Reply:
Nice catch. Going to be removing Zora Goerli the registry so no longer relevant.